CONTRIBUTING.md

```diff
@@ -6,6 +6,8 @@ Start by reading the [Engineering Design](./docs/InferenceHighLevelDesign.md). Y
 ## Proposing new public APIs
 
  ONNX Runtime has a collection of [public APIs](./README.md#api-documentation).  Some of these APIs make their way back into the Windows OS.  We make compatibility commitments for these APIs and follow a structured process when adding to them.  Please use the [Feature Request issue template](https://github.com/microsoft/onnxruntime/issues/new?template=feature_request.md) before starting any PRs that affect any of the public APIs.
+ 
+ If your proposal affects the Windows Machine Learning WinRT API, please review [Windows ML main principles](./docs/WinML_principles.md) before you submit your feature request. 
```

Comment:
should we call out what we mean by changing the "Windows Machine Learning WinRT API"? We could point to specific files, and say, "if you're proposing to change these files, please read this page first"

Reply:
we could also say "Windows.AI.MachineLearning" namespace.  that makes it clear what we mean.

---
In reply to: [505698266](https://github.com/microsoft/onnxruntime/pull/5453#discussion_r505698266) [](ancestors = 505698266)

Reply:
fixed